Token.js

```diff
@@ -2439,10 +2439,12 @@ function toggle_player_selectable(tokenInstance, token){
 	if (tokenInstance.options.locked && !window.DM){
 		tokenImage?.css("cursor","default");
 		tokenImage?.css("pointer-events","none");
+		token.css("pointer-events", "none");
```

Comment:
Would it be worth turning off other events too like our keybinds on it? Would save some listeners just sitting there

Reply:
we could - then add them back if unlocked. Just for players though DM's still need to access the context menu etc.

I think we can do this in another PR though. I'm going to merge this in for the fix for now.